src/stan/lang/ast/fun/is_nil_vis.hpp

```diff
@@ -36,6 +38,8 @@ namespace stan {
       bool operator()(const variable& x) const;  // NOLINT(runtime/explicit)
       bool operator()(const integrate_ode& x) const;  // NOLINT
       bool operator()(const integrate_ode_control& x) const;  // NOLINT
+      bool operator()(const algebra_solver& x) const;  // NOLINT
```

Comment:
I don't think these NOLINT markers are necessary, but don't worry about it for this pull request---not your problem.